src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -221,14 +223,20 @@ private void attachCaseByCcdId(
 
         ccdApi.attachExceptionRecord(
             theCase,
-            insertNewRecords(exceptionRecordDocuments, targetCaseDocuments),
+            buildCaseData(exceptionRecordDocuments, targetCaseDocuments),
             createEventSummary(theCase, exceptionRecordId, exceptionRecordDocuments),
             event
         );
 
         log.info("Attached exception record '{}' to case with CCD ID '{}'", exceptionRecordId, targetCaseCcdRef);
     }
 
+    private Map<String, Object> buildCaseData(List<Map<String, Object>> exceptionDocuments,
+                                              List<Map<String, Object>> existingDocuments) {
```

Comment:
let's use more often used (on this project at least) format:
```
functionName(
    Type1 param1,
    Type2 param2,
) {
```

Reply:
changed format